test/extended/router/OWNERS

```diff
@@ -1,14 +1,25 @@
 approvers:
-  - danehans
   - frobware
   - knobunc
   - Miciah
   - miheer
-  - sgreene570
+  - alebedev87
+  - candita
+  - gcs278
+  - rfredette
+  - Thealisyed
+  - grzpiotrowski
 reviewers:
-  - danehans
   - frobware
   - knobunc
   - Miciah
   - miheer
+  - alebedev87
+  - candita
+  - gcs278
+  - rfredette
+  - Thealisyed
+  - grzpiotrowski
+emeritus_approvers:
+  - danehans
```

Comment:
i think we dont this as he is not in the organization

Reply:
Ok but in the linked jira ticket, a comment was left to move the old owners to emeritus_approvers?

Reply:
yeah it looks like we need to keep him. No worries.